### PR TITLE
Calin/hotfix

### PR DESCRIPTION
**Pull request by calin-coj**
* updated endpoints signature
* made sure it integrates well with latest version of zia-canteen
* updated api description
* updated readme
* hosted swagger page: https://app.swaggerhub.com/apis/neurolabs/zia-server/2.0
* enforced payload validation

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] I am happy with the test coverage for the newly added code.
- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.

- [ ] `fixes #issue_number` or `closes #issue_number` in the body of the pull request.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,16 +7,8 @@ The software provided by Neurolabs acts as a backend for the partner Software In
...
-2) GET /checkout
-3) POST /menu
-
-### Description 
-* `GET /image` - Using the hardware camera, takes a picture and returns it 
-* `GET /checkout` - A picture is taken with the camer, and then  object detection is performed, providing a list of the products that were detected
-* `POST /checkout` - Saves a checkout.
-* `POST /menu` - Updates the menu. This endpoint is used to tell the Neurolabs software which of the products from the menu are going to be available for the current day
+### The API Description
+Check the [Swagger here](https://app.swaggerhub.com/apis/neurolabs/zia-server/2.0)
```
Comment (bnorbert) [inline README.md]: this is brilliant.

Comment (ppop) [reply README.md]: indeed :)

File: src/zia_connect/apis/namespace_catalog.py [hunk 2]
```diff
@@ -81,63 +102,74 @@ def get(self, canteen_id):
...
+        description=
+        "Returns the information about a specific product from the canteen")
     def get(self, canteen_id, id):
         """ Returns an item from the catalog NOT IMPLEMENTED
         """
 
         return mock_catalog_list[0]
 
-    @api.doc(responses={200: 'Success'})
+    @api.doc(responses={204: 'Success'})
```
Comment (ppop) [inline src/zia_connect/apis/namespace_catalog.py]: 204 is not success

File: src/zia_connect/apis/namespace_pos.py [hunk 3]
```diff
@@ -96,25 +109,35 @@ def post(self):
...
-                              img_height=request["img_height"])
-        
+        path = backend().save_checkout(
+            checkout_id=request["checkout_id"],
+            checkout_basket=ps.detection_frontend2backend(
+                request["detections"]),
+            img_width=request["img_width"],
+            img_height=request["img_height"])
+
         return "", 204
```
Comment (ppop) [inline src/zia_connect/apis/namespace_pos.py]: 204 != success

**Pull request merged**